test/test_torchscript_consistency.py

```diff
@@ -304,85 +304,75 @@ def test_perf_biquad_filtering(self):
         _test_torchscript_functional(F.lfilter, waveform, a, b)
 
 
-RUN_CUDA = torch.cuda.is_available()
-print("Run test with cuda:", RUN_CUDA)
+def _assert_transforms_consistency(transform, tensor, device):
```

Comment:
Does this make the interface different for functionals?

Reply:
Yes, functionals do not have `to` method, so Tensor arguments have to be moved to the target device manually.

Reply:
Would there be a way to keep the two implementations closer?

Reply:
Refer to my [WIP](https://github.com/mthrok/audio/commit/c736e21746653c1132654d80adb83e71dc96d757#diff-04757c68c2f80adf8dac82791cd9d5beR13). To me this is very close.